src/main/groovy/edu/oregonstate/mist/locations/db/ArcGisDAO.groovy

```diff
@@ -48,8 +50,11 @@ class ArcGisDAO {
         def mappedData = mapper.readTree(gisData).get("features")
         def data = new HashMap<String, GenderInclusiveRRLocation>()
 
-        if (mappedData.size() == 0) {
-            throw new DAOException("found zero gender inclusive restrooms")
+        int numFound = mappedData.size()
+
+        if (numFound < ARCGIS_THRESHOLD) {
```

Comment:
You have a lot of these checks which are pretty similar. They compare the size of two integers, and throw an exception if one is less than the other. You could probably put this check in a static shared method somewhere?